src/sdk/pynni/nni/compression/torch/pruning/amc/lib/net_measure.py

```diff
@@ -85,11 +85,11 @@ def measure_layer(layer, x):
     return
 
 
-def measure_model(model, H, W):
+def measure_model(model, H, W, device):
```

Comment:
where is this function used? you modified its definition but there is no corresponding modification in the calls of this function

Reply:
Thanks for good catch, fixed.